src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,44 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy.dao");
```

Comment:
Ensure that the `Injector` class is correctly implemented to support dependency injection for `BookDao`. If `Injector.getInstance()` or `injector.getInstance(BookDao.class)` is not implemented correctly, it may lead to runtime errors.